lib/perl/Genome/Model/CwlPipeline/Command/PrepForTransfer.pm

```diff
@@ -45,9 +45,10 @@ sub execute {
     if ($self->md5sum) {
         push @headers, 'md5sum';
     }
-    if (-d $self->directory) {
-        $self->fatal_message('Unable to use existing directory: '. $self->directory);
-    } else {
+    if (-e $self->directory && !(Genome::Sys->directory_is_empty($self->directory)) ) {
+        $self->fatal_message('Unable to use existing non-empty directory: '. $self->directory);
+    }
+    unless (-d $self->directory) {
```

Comment:
Do we want to worry about the case where `$self->directory` points to an existing non-directory?  (Amusingly `Genome::Sys->directory_is_empty` returns true for files.)

Reply:
I assumed that Genome::Sys->create_directory would fail if the existing non-directory where in the way. I guess that was a bad assumption?

Reply:
It will.  You'll get a weird error like:
```Runtime error: directory does not exist: myfile.txt at Genome/Sys.pm line 935.```

Reply:
that is a strange/ambiguous error. Do you think I should handle an existing non-directory here?

Reply:
To be user-friendly I'd want to handle these two cases:
* `-e $dir && !-d $dir`
* `-d $dir && !Genome::Sys->directory_is_empty($dir)`

`Genome::Sys->create_directory($dir)` is fine with the directory already existing.